packages/react-percy-test-framework/src/interfaces/getCommonInterface.js

```diff
@@ -25,15 +27,20 @@ export default function getCommonInterface(suites) {
       suites.unshift(suite);
       if (typeof fn === 'function') {
         await fn.call(suite);
-      } else {
-        throw new Error(`Suite "${suite.fullTitle()}" was defined but no callback was supplied.`);
       }
       suites.shift();
       return suite;
     },
     snapshot(title, options, fn) {
       const snapshot = new Snapshot(title, options, fn);
       suites[0].addSnapshot(snapshot);
+      const snapshotFullTitle = snapshot.fullTitle();
+      if (snapshotNames[snapshotFullTitle]) {
+        throw new Error(
+          `A snapshot named \`${snapshotFullTitle}\` has already been added, please use a different name`,
```

Comment:
Ideally this should include the path of the file in which the snapshot with this title was already defined so users can easily track it down, but that'll require a bit more work as this will need to be context-aware of what file it's in. Punting on that for now, as the way we do that will change when we switch to client-side rendering rather than server-side rendering. I'll circle back to this once I've done the client-side rendering work.